iam.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_iam_instance_profile" "default" {
+  name = module.this.id
+  role = module.this.id
+}
+
+resource "aws_iam_role" "default" {
+  name = module.this.id
+  path = "/"
+
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "sts:AssumeRole",
+            "Principal": {
+               "Service": "ec2.amazonaws.com"
+            },
+            "Effect": "Allow",
+            "Sid": ""
+        }
+    ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "ecs_execution_managed_policy" {
```

Comment:
instead of 3 separate instances of `aws_iam_role_policy_attachment`, can we use a single one with policy arns passed in and a `for_each` here ?